runtime/binding-kafka/src/test/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/ClientMergedIT.java

```diff
@@ -368,7 +368,7 @@ public void shouldFetchMergedMessagesWithHeadersSkipManyFilter() throws Exceptio
     @Specification({
         "${app}/merged.fetch.filter.sync/client",
         "${net}/unmerged.fetch.filter.sync/server"})
-    public void shouldFetchMergedMessageLatest() throws Exception
+    public void shouldFetchMergedLatestMessages() throws Exception
```

Comment:
Usually we name the test methods according to the spec scenarios.
In this case, i think that would be `shouldFetchMergedFilterSync`, works?